GPU/GPUTracking/Global/GPUChainTrackingClusterizer.cxx

```diff
@@ -665,7 +664,7 @@ int32_t GPUChainTracking::RunTPCClusterizer(bool synchronizeOutput)
       nnTimers[11] = &getTimer<GPUTPCNNClusterizer, 11>("GPUTPCNNClusterizer_ONNXRegression2_2_", 11);
     }
 
-    mRec->runParallelOuterLoop(doGPU, numLanes, [&](uint32_t lane) {
+    for (int32_t lane = 0; lane < numLanes; lane++) {
```

Comment:
Just curious about this change here. When running with GPU, it should not make any difference. But when running on CPU backend, the new version would serialize while the old version would run in parallel.
Is this change intended?